google-cloud-spanner-hibernate-dialect/src/main/java/com/google/cloud/spanner/hibernate/SpannerServiceContributor.java

```diff
@@ -32,10 +37,27 @@
  */
 public class SpannerServiceContributor implements ServiceContributor {
 
+  private static final SpannerSchemaManagementTool SCHEMA_MANAGEMENT_TOOL =
```

Comment:
I think we need to update the comment above to reflect the fact that this class is doing more than before.